integration/integration_test.go

```diff
@@ -30,25 +32,55 @@ func TestBuilder_Build(t *testing.T) {
 	if i.ProtocolVersion != "2" {
 		t.Error()
 	}
-	if len(i.Data) != 0 {
+	if len(i.Entities) != 0 {
 		t.Error()
 	}
 }
 
-func TestBuilder_BuildWithDefaultArguments(t *testing.T) {
+func TestDefaultIntegrationWritesToStdout(t *testing.T) {
+	// capture stdout to file
+	f, err := ioutil.TempFile("", "stdout")
```

Comment:
Why do you need to write to a file? Why not creating a simple buffer?

Reply:
I changed the rest of tests to a simple buffer, but this case is testing the STDOUT, so if you don't use it then you are not testing the SUT